### ISSUE TITLE
Is 'http://cubosphere.vallua.ru/artifacts' down?

### ISSUE THREAD

**Issue by SusDTSteve** [cubosphere/cubosphere-code#60]
**Describe the bug**
When ever I try to go onto the website with the git builds, the website returns with this error: DNS_PROBE_FINISHED_NXDOMAIN
Is anyone else experiencing this?

**To Reproduce**
1. Go to 'http://cubosphere.vallua.ru/artifacts'
2. Get the error

**Expected behavior**
Go on the website.

**Screenshots**

**Desktop (please complete the following information):**
 - OS:  Ubuntu Linux
 - Version NONE

**Comment by v1993** [comment]
Yeah, I'm not maintaining that website anymore and don't really plan to bring it back up.

**Comment by JappaWakka** [comment]
will there be a different way to download the game for windows?
